apps/samourai-server/docker-compose.yml

```diff
@@ -93,7 +93,7 @@ services:
          ipv4_address: $APP_SAMOURAI_SERVER_NODE_IP
 
   whirlpool:
-    image: louneskmt/dojo-whirlpool:1.2.1@sha256:8674bca0d901e8d65d49e5cf38c597c37bf1d99168114a58b63b242dd1b38d05
+    image: louneskmt/dojo-whirlpool:1.3.0
```

Comment:
Where do these version numbers come from? Because `louneskmt/dojo-whirlpool:1.3.0` includes Whirlpool CLI 0.10.9 which is a little confusing.

Can the container version reflect the version of the service it's packaging? Or is there a reason it's seperate?

Reply:
I use Dojo containers versions. 

https://code.samourai.io/dojo/samourai-dojo/-/blob/master/docker/my-dojo/.env#L21

Reply:
Outdated by 14e0350.